.github/workflows/hyperjump-comment.yml

```diff
@@ -0,0 +1,21 @@
+name: (hyperjump) comment
+
+on:
+  repository_dispatch:
+    types: [comment]
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    name: (hyperjump) comment
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: comment
+        uses: ./.github/actions/hyperjump-comment
```

Comment:
sweeeet! It looks like we can do the same in LBT route its comment through `hyperjump`